modules/core/src/test/java/org/apache/ignite/testframework/junits/IgniteConfigVariationsAbstractTest.java

```diff
@@ -92,22 +73,17 @@
     @Override public String getTestIgniteInstanceName() {
         return "testGrid";
     }
-
-    /**
-     * Invoked by reflection from {@code ConfigVariationsTestSuiteBuilder}.
```

Comment:
I think that we can add a comment to `testsCfg` fields stating that it is assigned by generated code.

Reply:
Ok, I paid attention to this moment in comments